health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/components/HCMMyCampaignRowCard.js

```
@@ -359,6 +405,24 @@
+            <div>
+              <CardText style={{ margin: 0 }}>{t("RETRY_IN_PROGRESS")}</CardText>
+            </div>,
+          ]}
+          onOverlayClick={() => {
+            setShowRetryPopUp(false);
+          }}
+          onClose={() => {
+            setShowRetryPopUp(false);
+          }}
+          footerChildren={[]}
+        ></PopUp>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)